patterns/footer-large.php

```diff
@@ -29,11 +29,15 @@
 <!-- /wp:column -->
 
 <!-- wp:column {"width":"20%","style":{"spacing":{"padding":{"top":"0px"}}}} -->
-<div class="wp-block-column" style="padding-top:0px;flex-basis:20%"><!-- wp:navigation {"layout":{"type":"flex","orientation":"vertical"}} /--></div>
+<div class="wp-block-column" style="padding-top:0px;flex-basis:20%"><!-- wp:navigation {"layout":{"type":"flex","orientation":"vertical"}} -->
+<!-- wp:page-list /-->
+<!-- /wp:navigation --></div>
```

Comment:
I think this should go back to not have the `wp:page-list` block:
```suggestion
<div class="wp-block-column" style="padding-top:0px;flex-basis:20%"><!-- wp:navigation {"layout":{"type":"flex","orientation":"vertical"}} /--></div>
```

(There are several other instances of this in other light patterns as well)